networking_nsxv3/api/rpc.py

```diff
@@ -231,69 +170,82 @@ def plugin(self):
         return self._plugin
 
     @log_helpers.log_method_call
-    def get_port_revision_tuples(self, context, host, limit, created_after):
-        return db.get_port_revision_tuples(context, host, limit, created_after)
+    def get_ports_revisions(self, context, host, limit, offset):
+        return db.get_ports_revisions(context, host, limit, offset)
 
     @log_helpers.log_method_call
-    def get_qos_policy_revision_tuples(self, context, limit, created_after):
-        return db.get_qos_policy_revision_tuples(context, limit, created_after)
-
+    def get_qoses_revisions(self, context, host, limit, offset):
```

Comment:
`qoses` -> `qos_policies`
Relationship of ports and revisions unclear from name.